xbmc/cores/AudioEngine/CMakeLists.txt

```diff
@@ -95,27 +95,28 @@ if(CORE_SYSTEM_NAME STREQUAL rbpi)
                       Sinks/AESinkPi.h)
 endif()
 
-if(CORE_SYSTEM_NAME STREQUAL darwin OR CORE_SYSTEM_NAME STREQUAL ios)
-  list(APPEND SOURCES Sinks/osx/AEDeviceEnumerationOSX.cpp
+if(CORE_SYSTEM_NAME STREQUAL darwin)
```

Comment:
`CMAKE_SYSTEM_NAME` is Darwin for all Apple platforms (osx, ios, tvos).
`CORE_SYSTEM_NAME` is darwin for osx but it's set to ios for ios. Memphiz already told me that we should change `CORE_SYSTEM_NAME` to osx at some point to be consistent. Here we really mean `if(osx)` hence it should be correct the way it is.